pkg/apiserver/rest/apis/v1/types.go

```diff
@@ -104,6 +104,8 @@ type ListAddonRegistryResponse struct {
 type EnableAddonRequest struct {
 	// Args is the key-value environment variables, e.g. AK/SK credentials.
 	Args map[string]interface{} `json:"args,omitempty"`
+	// Clusters specify the clusters this addon should be installed, if not specified, it will follow the configure in addon metadata.yaml
+	Clusters []string `json:"clusters,omitempty"`
```

Comment:
Will there be any addons that need to be installed in a different namespace in managed clusters?

Reply:
good question, currently, users define namespace in the component spec by using raw type component